components/function-runtimes/python38/Dockerfile

```diff
@@ -9,6 +9,8 @@
 
 FROM alpine:3.14.2
 
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
```

Comment:
ash?

Reply:
It's the shell for alpine. I'm following the recommendation from hadolint tool https://github.com/hadolint/hadolint/wiki/DL4006#set-the-shell-option--o-pipefail-before-run-with-a-pipe-in.